minutes/_project/README.md

```diff
@@ -0,0 +1,9 @@
+The "Project" meetings (formerly known as CSSI meetings) focus on PlasmaPy project
```

Comment:
Ah!  I was thinking something like this in my last comment, which I'm feeling too sleepy to change.